biothings/web/query/formatter.py

```diff
@@ -191,7 +208,23 @@ def transform(self, response, **options):
                         hit_.update(template_hit)
                         hit_.update(hit)
                         responses_.append(hit_)
-            return list(filter(None, responses_))
+            response_ = list(filter(None, responses_))
+            if options.with_total:
+                response_ = {
+                    'max_total': max_total,
+                    'hits': response_,
+                }
+                max_size = options.size or 1000
```

Comment:
is `option.size` default 1000 or just 10?

Reply:
I think it is 1000. Not 100% sure. Because I don't find out the default value in case user not pass the size option.
If there is an other maximum size, please let's me know :)

Reply:
@sengineer0 can you please find out? You can capture the raw queries sent to ES, with or without `size` parameter to see how `size` was passed to ES eventually. It's important to justify your hard-coded `1000` here. If confirmed, some comments are needed here to explain why 1000 here.

Reply:
oh, I didn't know that size is restrict by ES or the ES's client. Let's me check it, and I will answer you later :)

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html#search-multi-search-api-request-body

as the ES's documentation noted, the default size will be 10. I'll update my code to this value.

Reply:
it's updated.
@newgene